src/BloomBrowserUI/bookEdit/pageThumbnailList/pageThumbnailList.less

```diff
@@ -32,25 +32,25 @@ BODY {
         height: 65px; width: 47px; /* default = A5Portrait */
 
         &.A5Landscape {
-                height: 51px;
-                width: 69px;
-                .bloom-page {
-                        transform: scale(0.08);
-                }
+            height: 51px;
```

Comment:
If this is just auto formatting, please don't include it in your PR. Feel free to commit directly to master.